fastlane/Fastfile

```diff
@@ -13,209 +13,174 @@
 # update_fastlane
 
+skip_docs
+
 default_platform(:ios)
 
 # CONFIG VARIABLES
-app_name = 'ios-base'
-username = '' # The apple id that you are using to admin the certificates eg: toptiertest@gmail.com
-xcodeproj = 'ios-base.xcodeproj' # The xcodeproj name eg: ReactNativeBase.xcodeproj
-workspace =  'ios-base.xcworkspace' # The xworkspace name here eg: ReactNativeBase.xcworkspace
-certificates_git_url = 'git@github.com:rootstrap/ios-base-certificates.git' # The repo to store and sync certs and provisioning profiles. Eg: git@github.com:rootstrap/cafe-infinity-certificates.git
-team_id = ''  # The organization's team id (you can leave this empty and select it on the CLI on each run)
-itc_team_id = '' # App Store Connect team id
-distribute_external = false # Change this to true if you want the build to be distributed to external testers as well.
-notify_external_testers = false
-skip_waiting_for_build_processing = true # Don't wait for the build to process. If set to true, the changelog won't be set, distribute_external option won't work and no build will be distributed to testers. (You might want to use this option if you are using this action on CI and have to pay for 'minutes used' on your CI plan)
-devices = 'platform=iOS Simulator,OS=9.2' # Replace with proper array of devices to run the tests on
-
-platform :ios do
-  
-  lane :release_develop do
-    sync_code_signing(
-      git_url: certificates_git_url,
-      username: username,
-      type: 'appstore',
-      team_id: team_id
-    )
+app_name              = 'ios-base'
+username              = ENV["FASTLANE_USER"]                                  # The apple id that you are using to admin the certificates eg: toptiertest@gmail.com
+xcodeproj             = 'ios-base.xcodeproj'                                  # The .xcodeproj file name
+workspace             = 'ios-base.xcworkspace'                                # The xworkspace file name
+certificates_git_url  = 'git@github.com:rootstrap/ios-base-certificates.git'  # The repo to store and sync certs and provisioning profiles.
+team_id               = ENV["FASTLANE_TEAM_ID"]                               # The organization's team id 
+itc_team_id           = ENV["FASTLANE_ITC_TEAM_ID"]                           # App Store Connect team id
+distribute_external   = false                                                 # Change this to true if you want the build to be distributed to external testers as well.
+notify_testers        = false                                                 #
+skip_wait             = true                                                  # Don't wait for the build to process. If set to true, the changelog won't be set, distribute_external option won't work and no build will be distributed to testers.
+devices               = 'platform=iOS Simulator,OS=9.2'                       # If using test lane, replace with proper array of devices to run the tests on
```

Comment:
Right now the ios-base minimum os target is 11.4 
You can change this to `OS=11.4` in order to match that configuration. 

We don't develop for poor people now (kind of) 😆 we still support SE gen 1 tho

Reply:
done :)